Build fails on windows

I have a fix will do a pull request.

```
npm ERR! prepareGitDep 1>
npm ERR! prepareGitDep > @mliebelt/pgn-parser@1.3.4 prepare C:\Users\mhobb\AppData\Roaming\npm-cache\_cacache\tmp\git-clone-a100e280
npm ERR! prepareGitDep > npm run build && npm test
npm ERR! prepareGitDep
npm ERR! prepareGitDep
npm ERR! prepareGitDep > @mliebelt/pgn-parser@1.3.4 build C:\Users\mhobb\AppData\Roaming\npm-cache\_cacache\tmp\git-clone-a100e280
npm ERR! prepareGitDep > node_modules/peggy/bin/peggy --allowed-start-rules pgn,tags,game,games -o _pgn-parser.js src/pgn-rules.pegjs
npm ERR! prepareGitDep
npm ERR! prepareGitDep
npm ERR! prepareGitDep 2> npm WARN install Usage of the `--dev` option is deprecated. Use `--also=dev` instead.
npm ERR! prepareGitDep 'node_modules' is not recognized as an internal or external command,
npm ERR! prepareGitDep operable program or batch file.
npm ERR! prepareGitDep npm ERR! code ELIFECYCLE
npm ERR! prepareGitDep npm ERR! errno 1
npm ERR! prepareGitDep npm ERR! @mliebelt/pgn-parser@1.3.4 build: `node_modules/peggy/bin/peggy --allowed-start-rules pgn,tags,game,games -o _pgn-parser.js src/pgn-rules.pegjs`
npm ERR! prepareGitDep npm ERR! Exit status 1
```

--- a/build.js
+++ b/build.js
@@ -1,0 +1,6 @@
+const path = require('path')
+const shell = require('shelljs')
+
+const peggyPath = path.resolve('./node_modules/peggy/bin/peggy')
+// node_modules/peggy/bin/peggy --allowed-start-rules pgn,tags,game,games -o _pgn-parser.js src/pgn-rules.pegjs
+shell.exec(`node ${peggyPath} --allowed-start-rules pgn,tags,game,games -o _pgn-parser.js src/pgn-rules.pegjs`)

--- a/package.json
+++ b/package.json
@@ -24,10 +24,11 @@
   "devDependencies": {
     "mocha": "^9.0.0",
     "peggy": "^1.2.0",
+    "shelljs": "^0.8.4",
     "should": "^13.2.3"
   },
   "scripts": {
-    "build": "node_modules/peggy/bin/peggy --allowed-start-rules pgn,tags,game,games -o _pgn-parser.js src/pgn-rules.pegjs",
+    "build": "node build.js",
     "test": "mocha",
     "test-interactive": "mocha -w",
     "prepare": "npm run build && npm test"

